tests/testthat/test-curl.R

```diff
@@ -204,11 +204,16 @@ test_that("can read from clipboard", {
   rlang::local_interactive()
 
   clipr::write_clip("curl 'http://example.com' \\\n -H 'A: 1' \\\n -H 'B: 2'")
-  expect_snapshot({
-    curl_translate()
-    # also writes to clip
-    clipr::read_clip()
-  })
+  expect_snapshot(
+    {
+      curl_translate()
+      # also writes to clip
+      clipr::read_clip()
+    },
+    transform = function(x) {
+      grep("\\\\r", x, value = TRUE, invert = TRUE)
+    }
```

Comment:
What's this for? Because I have had some sense that testthat was missing something, but the last time I looked into it, I couldn't find a problem: https://github.com/r-lib/testthat/pull/2004

Reply:
It's specifically that the *clipboard* contains a `\r` character. I'm pushing a more precise transform that I realized while working out an explanation here.

Reply:
I can't decide how I "wish" it worked. Maybe `clipr::read_clip()` should scrub `\r` line endings? Or maybe it's fine as-is and that might cause other issues?

Reply:
Right, but `expect_snapshot()` is supposed to strip all `\r`. So why does it matter?

Reply:
I'd have to dig in and walk through to grok it for sure, but it feels like this is a case where the vector literally includes the "\r" character, not that the "\r" is added on a file that's read or that sort of thing. I *think* I'd be confused if I had a vector with a carriage return character and testthat silently stripped it. I also feel like there's no REAL difference there, though, so I'm not certain that it's right for it to make it through.

Reply:
And I guess to clarify... It wasn't stripping the \r, so I had to add this for tests to pass. I made sure everything was updated in case it was that.

Reply:
Lets explore this in a separate PR, because I think this is a testthat bug.

Reply:
Fair enough, removing this change.

Reply:
See #669.